Bottom: 90aa7db616e01e9a86b79a52c9c743a731e33042
Top:    8f3d571aeb8b5b270df2c258a3d06d3e3847efce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 23:52:20 +0100

Refresh of hid-gerber-don-t-use-the-core-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 003a77e..af3ad50 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -502,8 +502,7 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
         continue;
 
       if (set_layer (0, group))
-        if (DrawLayerGroup (group, drawn_area))
-          DrawPPV (group, drawn_area);
+        DrawLayerGroup (group, drawn_area);
     }
 
   count_holes (drawn_area, &nplated, &nunplated);
